packages/components/psammead-navigation/src/DropdownNavigation/index.jsx

```diff
@@ -156,6 +155,7 @@ const MenuButton = styled(Button)`
       ${iconBorder};
     }
   }
+  border: 0;
```

Comment:
Can you put this style after the `background-color: transparent;`?

Reply:
Done